Example/SlideController/PageLifeCycleObject.swift

```diff
@@ -37,17 +37,17 @@ extension PageScrollViewLifeCycle_Implementation : PageScrollViewLifeCycle {
         
     }
     
-    func didStartScrolling() {
+    func didStartSliding() {
         
     }
     
-    func didCancelScrolling() {
+    func didCancelSliding() {
         
     }
 }
 
 private typealias Viewable_Implementation = PageLifeCycleObject
-extension Viewable_Implementation : SlideController.Viewable {
+extension Viewable_Implementation : Viewable {
```

Comment:
`+extension Viewable_Implementation : Viewable {` looks same with `ViewAccessible`. Do we really ween two same protocols?

Reply:
ViewAccessible is protocol for Example app. Viewable is public protocol defined in SlideController. Usage ViewAccessible in Example classes not required, it is something I did to follow the same structure.